### PR TITLE
Display special report label

### PR DESCRIPTION
**Pull request by alexflorisca**
🐿 v2.5.14

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ bower_components/
 /public/main.css
 /pa11y_screenCapture/*
 *.env*
+.idea/
```
Comment (wheresrhys) [inline .gitignore]: could you put this in your global .gitignore - we've had problems in the past (still not totally resolved) with .gitignores in repos full of lots of text editor specific lines

**Comment by wheresrhys** [comment]
@adgad any thoughts on this PR - it's intended for native ads special reports, but is there a risk of it leaking out elsewhere? Shoudl the naming be more explicit that it's for native ads only?

**Comment by alexflorisca** [comment]
FIY, this is what the end result looks like. It just adds the "Special Report" label, but yes, I've just seem another teaser with a Special Report tag plus the title of the special report. Which comes from somewhere else I assume.

screen shot 2017-09-14 at 16 33 11

**Comment by adgad** [comment]
IIRC what we used to have as 'Special Report' as a genre tag (in the lighter font), and then the ad comes with a name/URL for the tag.

Do we just need to update the code here to be `teaserConcept` and `genreConcept`? 
https://github.com/Financial-Times/n-native-ads/blob/master/lib/format-response.js#L41

**Comment by alexflorisca** [comment]
@adgad Take a look at this and https://github.com/Financial-Times/n-teaser/pull/144/files and let me know if it's all good. Will leave it until Monday.

**Comment by adgad** [comment]
LGTM, just check out the front page to see if anything looks off 👍

**Pull request merged**